lms/djangoapps/certificates/management/commands/purge_pii_from_generatedcertificates.py

```diff
@@ -0,0 +1,45 @@
+"""
+A management command, designed to be run once by Open edX Operators, to obfuscate learner PII from the
+`Certificates_GeneratedCertificate` table that should have been purged during learner retirement.
+
+A fix has been included in the retirement pipeline to properly purge this data during learner retirement. This can be
+used to purge PII from accounts that have already been retired.
+"""
+
+import logging
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from lms.djangoapps.certificates.models import GeneratedCertificate
+from openedx.core.djangoapps.user_api.api import get_retired_user_ids
+
+User = get_user_model()
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    This management command performs a bulk update on `GeneratedCertificate` instances. This means that it will not
+    invoke the custom save() function defined as part of the `GeneratedCertificate` model, and thus will not emit any
+    Django signals throughout the system after the update occurs. This is desired behavior. We are using this
+    management command to purge leftover PII in the monolith, and we don't need updates to propogate to external systems
```

Comment:
This is a good comment, but "in the monolith" isn't clear for non-2U folks. How about something like:

```suggestion
    management command to purge remnant PII, retired elsewhere in the system, that should have already been removed from the Certificates tables. We don't need updates to propogate to external systems
```

Reply:
Thanks for the wordsmithing. I get what you mean, I'll make this change :)